Automattic/wp-calypso
# Components: Update TimeSince to use Interval; remove obsolete Ticker

Seems like the only usage of the legacy `Ticker` is in `<TimeSince />`. Ticker uses the legacy event emitter, so ideally we should get rid of it.

This PR updates that usage to `Interval`, which is a more widely used component for this purpose. Also, this PR suggests deleting the no longer used Ticker component.

This introduces a small behavioral change: `<TimeSince />` will now update regardless of whether it's currently in the viewport.

A potential future improvement could be to update it only when it's in the viewport. To achieve this, we could take advantage of the `visibilitychange` event in the `<Interval />` component.

#### Changes proposed in this Pull Request

* Update `<TimeSince />` to use the `<Interval />` component instead of `Ticker`.
* Delete the legacy `Ticker`, as it's no longer used.

#### Testing instructions

* Checkout this branch.
* Go to `http://calypso.localhost:3000/devdocs/design/time-since`
* Observe updates every 10 seconds.
* Verify the component works like it did before (except for the out-of-viewport case).
